ui/app/templates/components/mfa/mfa-login-enforcement-form.hbs

```diff
@@ -106,7 +106,8 @@
             @placeholder="Search for an existing target"
             @options={{this.searchSelect.options}}
```

Comment:
curious about using `@models={{array this.selectedTargetType}}` instead - which I _think_ would clear up the `.js` file for this form?